pages/workspaces/new.tsx

```diff
@@ -118,6 +120,9 @@ const NewWorkspace = () => {
             className="flex gap-2.5 items-center cursor-pointer w-min mt-2 sm:mt-0 self-end"
             loading={isSaving}
             loadingText={"Creating Workspace..."}
+            onClick={() => {
+              posthog.capture("clicked: finish Create Workspace");
```

Comment:
consider putting this event in the actual form submit that the button submits to as they can submit without clicking the button.

https://github.com/open-sauced/app/blob/8eb580dd723f81ef169af26830e74f2ed2a1fa42/pages/workspaces/new.tsx#L72-L98